cmd/rancher-desktop-guestagent/main.go

```diff
@@ -26,6 +26,7 @@ import (
 	"time"
 
 	"github.com/Masterminds/log-go"
+	containderd "github.com/rancher-sandbox/rancher-desktop-agent/pkg/containerd"
```

Comment:
```suggestion
	"github.com/rancher-sandbox/rancher-desktop-agent/pkg/containerd"
```
We should be less derd